src/interfaces.ts

```diff
@@ -270,7 +270,8 @@ export interface LoaderOptions {
   getCustomTransformers:
     | string
     | ((
-        program: typescript.Program
+        program: typescript.Program,
+        getProgram: () => typescript.Program
```

Comment:
Is it worth having `program` and `getProgram` given that `getProgram` returns a `program`?

Reply:
I just went with the suggestion @andersekdahl had in #1216. Removing the program in a minor or patch would be a breaking change...

Reply:
Most users will not have a problem with the original implementation, since most transformers don't rely on the sourceFiles in the program.

Reply:
Makes sense - thanks!